### PR TITLE
Reformat output or variant delete command and add analysis date, track and institute

### PR DESCRIPTION
**Pull request by northwestwitch**
Print delete variant command output and a \t separated table ad and add to the output:
- analysis date
- instritute
- case track

**How to test**:
1. Load some demo cases locally: demo case + cancer case?
1. Run the scout delete variants command with preferred parameters
1. Check that output contains a header and one line for each case with removed variants. Example:
`scout --demo --loglevel WARNING delete variants -u clark.kent@mail.com  -status inactive -rank-threshold 10 -variants-threshold 1 --dry-run`

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

File: scout/commands/delete/delete_command.py [hunk 1]
```diff
@@ -86,13 +87,18 @@ def variants(
             # Just print how many variants would be removed for this case
             remove_n_variants = store.variant_collection.count_documents(variants_query)
             total_deleted += remove_n_variants
-            click.echo(f"Remove {remove_n_variants} / {case_n_variants} total variants")
+            click.echo(
+                f"{nr}\t{n_cases}\t{case['owner']}\t{case['display_name']}\t{case_id}\t{case.get('track','')}\t{case['analysis_date']}\t{case_n_variants}\t{total_deleted}"
```
Comment (dnil) [inline scout/commands/delete/delete_command.py:91]: 😊 Whats wrong with "\t".join(list)?

**Review by dnil — APPROVED** [review]

File: scout/commands/delete/delete_command.py (hunk 1 above)
Comment (northwestwitch) [reply scout/commands/delete/delete_command.py:91]: Nothing :)

File: scout/commands/delete/delete_command.py [hunk 2]
```diff
@@ -21,7 +22,7 @@
     help="Restrict to cases with specified status",
 )
 @click.option("-older-than", type=click.INT, default=0, help="Older than (months)")
-@click.option("-rank-threshold", type=click.INT, default=5, help="With rank threshold lower than")
+@click.option("-rank-threshold", type=click.INT, default=10, help="With rank threshold lower than")
```
Comment (northwestwitch) [inline scout/commands/delete/delete_command.py:25]: @dnil what about this?

**Comment by northwestwitch** [comment]
tested and it works:

image

File: scout/commands/delete/delete_command.py (hunk 2 above)
Comment (dnil) [reply scout/commands/delete/delete_command.py:25]: I liked 5, only missed the default before.. 😄

Comment (northwestwitch) [reply scout/commands/delete/delete_command.py:25]: Ok. I'll leave it as it was, doesn't matter really.

**Pull request merged**